src/plays/git-hub-profile-search/Readme.md

```diff
@@ -0,0 +1,27 @@
+# GitHub Profile Search
+
+A play in which when you type someone's github username it displays there public github profile.
+
+## Play Demographic
+
+- Language: js
+- Level: Beginner
+
+## Creator Information
+
+- User: nirban256
+- Gihub Link: https://github.com/nirban256
+- Blog: null
+- Video: null
+
```

Comment:
There is a bug that these values are coming as null instead of NA. Please correct them manually.

Reply:
@atapas if remove these fields will the bug get removed?

Reply:
No it will just fix for ur play. You can replace the null with NA

Reply:
ok